test/integration/narrow/routes/disabled.test.js

```diff
@@ -0,0 +1,46 @@
+const config = require('../../../../app/config')
+config.endpointEnabled = false
```

Comment:
consider move into `beforeEach` as tht's where state and changed params etc are usually set

Reply:
Done, only reason I didn't was would mean require would have to move too, but I guess that's no big deal.  Moved as suggested.

Reply:
We can have `require` outside, no?